packages/components/src/form-token-field/types.ts

```diff
@@ -182,6 +182,12 @@ export interface FormTokenFieldProps
 	 * @default false
 	 */
 	__nextHasNoMarginBottom?: boolean;
+	/**
+	 * If true add any incompleteTokenValue as a new token.
```

Comment:
```suggestion
	 * If true, add any incompleteTokenValue as a new token when the field loses focus.
```

Small typo tweak and making this consistent with the readme.

Reply:
fixed.